static/js/components/capability/smart-plug.js

```diff
@@ -62,12 +62,19 @@ class SmartPlugCapability extends BaseComponent {
 
     this._icon = this.shadowRoot.querySelector('#icon');
     this._label = this.shadowRoot.querySelector('#label');
+
+    this._havePower = false;
     this._on = false;
     this._power = 0;
+
     this._onClick = this.__onClick.bind(this);
   }
 
   connectedCallback() {
+    this._havePower =
+      typeof this.dataset.havePower !== 'undefined' ?
+        this.dataset.havePower :
```

Comment:
If someone specifies `data-have-power="false"` this evaluates to `"false"` which is truthy (oh Javascript, what a beautiful language)